src/Webtoons/WebtoonsParser.ts

```diff
@@ -0,0 +1,380 @@
+import { Chapter, ChapterDetails, HomeSection, LanguageCode, Manga, MangaStatus, MangaTile, PagedResults } from "paperback-extensions-common";
+import { PagedSearchResults } from "./WebtoonHelperStructures";
+const WEBTOON_DOMAIN = 'https://www.webtoons.com'
+
+export const parseMangaDetailsChallenge = async ($: CheerioStatic, mangaId: string): Promise<Manga> => {
+const title = $('h3._challengeTitle').text().replace('DASHBOARD', '').trim()
+
+const image = $('img', $('span.thmb')).attr('src')
+
+if(!image) {
+    throw new Error(`Failed to parse MangaDetails challenge image for ${mangaId}`)
+}
+
+let rating = $('em#_starScoreAverage').text()
+const completed = $('span.txt_ico_completed2') === undefined ? MangaStatus.ONGOING : MangaStatus.COMPLETED
```

Comment:
```suggestion
const completed = title.toLowerCase().includes('completed') ? MangaStatus.COMPLETED : MangaStatus.ONGOING
```
Completed manhwa are marked in title